src/index.js

```diff
@@ -339,7 +360,9 @@ class AvatarEditor extends React.Component {
   }
 
   componentWillReceiveProps (newProps) {
-    if (newProps.image && this.props.image !== newProps.image) {
+    if ((newProps.image && this.props.image !== newProps.image) ||
```

Comment:
I guess it's not an expected use case for the image width & height to be updated after the component has rendered, but currently it doesn't work (in many situations the image needs rescaling to fit the new canvas size).  To fix this we can just view changes to the width and height the same way as changes to the image itself.  It's not as performant as it could be, but if I'm correct in my supposition that these properties aren't going to change at runtime, then it seems like a reasonable tradeoff.  It also makes the demo work :)